### ISSUE TITLE
Create Proper Landing page for the website

### ISSUE THREAD

**Issue by neutr0n420** [Rise-In/NFT-Minter#7]
Right now the application don't have any landing page, Create a attractive landing page for the application with Responsive Design.
the Landing page should include
- [ ] A Proper Navbar
- [ ] A proper Header
- [ ] A section about Rise In
- [ ] Footer
Current preview for the landing page looks something like this 👇🏻
<img width="1398" alt="Screenshot 2024-10-18 at 1 10 26 PM" src="https://github.com/user-attachments/assets/ec9615b7-41c2-4ef6-82ea-3b0e9183464d">
tldr: We can use UI library like ShadCN
